documentation/asciidoc/topics/con_cross_site_expose_types.adoc

```diff
@@ -15,12 +15,25 @@ In most cases, an administrator must also configure DNS routing for those extern
 
 .`LoadBalancer`
 
-A `LoadBalancer` is service that directs network traffic to the correct node in the {openshiftshort} cluster.
+A `LoadBalancer` is a service that directs network traffic to the correct node in the {openshiftshort} cluster.
 
 Whether you can use a `LoadBalancer` as the expose type for cross-site replication depends on the host platform.
 AWS supports network load balancers (NLB) while some other cloud platforms do not.
 To use a `LoadBalancer` service, an administrator must first create an ingress controller backed by an NLB.
 
+.`Route`
+
+An {openshiftshort} `Route` allows {brandname} clusters to connect with each other through a secure URL.
+
+{brandname} uses TLS with the SNI header to send backup requests between clusters through an {openshiftshort} `Route`.
+To do this you must add a keystore so that {brandname} can encrypt RELAY messages.
+
+When you specify `Route` as the expose type for cross-site replication, {ispn_operator} creates a routes with TLS passthrough encryption for each {brandname} cluster that it manages.
+You can specify a hostname for the `Route` but you cannot specify a `Route` that you have already created.
+ifdef::community[]
+Likewise it is not possible to use Kubernetes ingress instead of a route.
```

Comment:
is it worth mentioning that Kubernetes does not support TLS + SNI and that's why it does not work?